ivy_tests/test_ivy/test_frontends/test_tensorflow/test_metrics.py

```diff
@@ -860,3 +918,85 @@ def test_tensorflow_cosine_similarity(
         y_true=y_true,
         y_pred=y_pred,
     )
+
+
+# deserialize
+@handle_frontend_test(
+    fn_tree="tensorflow.keras.metrics.deserialize",
+    fn_name=st.sampled_from(get_callable_functions("keras.activations")).filter(
+        lambda x: not x[0].isupper()
+        and x
+        not in [
+            "deserialize",
+            "get",
+            "keras_export",
+            "serialize",
+            "deserialize_keras_object",
+            "serialize_keras_object",
+            "get_globals",
+        ]
+    ),
+    dtype_and_data=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_value=0,
+        max_value=10,
+        shape=helpers.ints(min_value=2, max_value=5).map(lambda x: tuple([x, x])),
+    ),
+)
+def test_tensorflow_deserialize(
+    *,
+    dtype_and_data,
+    fn_name,
+    fn_tree,
+    frontend,
+):
+    dtype_data, data = dtype_and_data
+    simple_test_two_function(
+        fn_name=fn_name,
+        x=data[0],
+        frontend=frontend,
+        fn_str="deserialize",
+        dtype_data=dtype_data[0],
+        rtol_=1e-01,
+        atol_=1e-01,
+        ivy_submodules=["keras", "metrics"],
+        framework_submodules=["keras", "activations"],
+    )
+
+
+# get 
+@handle_frontend_test(
+    fn_tree="tensorflow.keras.metrics.get",
+    fn_name=st.sampled_from(get_callable_functions("keras.activations")).filter(
+        lambda x: not x[0].isupper()
+        and x
+        not in [
+            "deserialize",
+            "get",
+            "keras_export",
+            "serialize",
+            "deserialize_keras_object",
+            "serialize_keras_object",
+            "get_globals",
+        ]
+    ),
+    dtype_and_data=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=0,
+        max_value=10,
+        shape=helpers.ints(min_value=2, max_value=5).map(lambda x: tuple([x, x])),
+    ),
+)
+def test_tensorflow_get(fn_name, dtype_and_data):
+    dtype_data, data = dtype_and_data
+    simple_test_two_function(
+        fn_name=fn_name,
+        x=data[0],
+        frontend="tensorflow",
+        fn_str="get",
+        dtype_data=dtype_data[0],
+        rtol_=1e-01,
+        atol_=1e-01,
+        ivy_submodules=["keras", "metrics"],
+        framework_submodules=["keras", "activations"],

```

Comment:
framework submodules should be `["keras", "metric"]`

Reply:
if I change the `activations` to `metrics`, it results in all tests being failed...